lib/Models/WebMapServiceCatalogItem.js

```diff
@@ -1279,20 +1284,6 @@ function updateIntervalsFromIsoSegments(intervals, isoSegments, time, wmsItem) {
     }
 }
 
-function formatMomentForWms(m, duration) {
-    // If the original moment only contained a date (not a time), and the
-    // duration doesn't include hours, minutes, or seconds, format as a date
-    // only instead of a date+time.  Some WMS servers get confused when
-    // you add a time on them.
-    if (duration.hours() > 0 || duration.minutes() > 0 || duration.seconds() > 0 || duration.milliseconds() > 0) {
```

Comment:
This logic is important and needs to be preserved.

Reply:
noted. Think it would be good to add a test around this somehow